target/linux/ath79/generic/base-files/etc/uci-defaults/04_led_migration

```diff
@@ -29,6 +29,11 @@ tplink,re355-v1)
 tplink,re450-v1)
 	migrate_leds "re450:=tp-link:"
 	;;
+tplink,tl-wdr3600-v1|\
+tplink,tl-wdr4300-v1|\
+tplink,tl-wdr4300-v1-il)
+	migrate_leds ":blue:=:green:"
```

Comment:
It's great you spotted this, +1.

Reply:
Also, perhaps you could merge that into archer-c7-v2 case.

Reply:
Oh, I didn't noticed that. Merged.